### PR TITLE
Adding LT language

### PR DESCRIPTION
**Pull request by tomaziss**

### CODE REVIEW

**Comment by Lowentwickler** [comment]
ačiū :)

Looks good to me. CC: @ondratu

File: src/locales/source/lt.json [hunk 1]
```diff
@@ -0,0 +1,258 @@
+{
+  "btn.atgal": "Atgal",
```
Comment (ondratu) [inline src/locales/source/lt.json:2]: The key have to stay original, only value can be translated.

**Review by ondratu — CHANGES_REQUESTED** [review]
Thank you a lot for your work. But you was translate keys too, which is bad. You have to translate only values, the keys are used to right paring :-)

File: src/locales/source/lt.json (hunk 1 above)
Comment (tomaziss) [reply src/locales/source/lt.json]: Reverted key translations...

**Review by ondratu — CHANGES_REQUESTED** [review]
Great! Thank you much! Could you rebase our master branch please, at this moment the pull request look worse cause GitHub ...

**Pull request closed without merging**